docs/operation.md

```diff
@@ -75,7 +75,7 @@ The `Sender email` and `Receiver email` each identify a different protected enti
 
 > **This version of __Diffix for Desktop__ does not protect a data set where there are multiple protected entities per row**
 
-A data set with multiple protected entities needs to be pre-processed to have one protected entity per row before loading into __Diffix for Desktop__. See [Multiple protected entities per row](#multiple-protected-entities-per-row).
+A data set with multiple protected entities needs to be pre-processed to have one protected entity per row before loading into __Diffix for Desktop__. See [Multiple protected entities per row](/tips.md#multiple-protected-entities-per-row).
```

Comment:
Does the leading slash work on github as well? Maybe it's safer to drop it?

Reply:
oh right, it works without `/`. I got this slash from reading the links components code, but I think I misunderstood it on quick scan. Fixed